Client/mods/deathmatch/logic/luadefs/CLuaEffectDefs.h

```diff
@@ -37,4 +37,6 @@ class CLuaEffectDefs : public CLuaDefs
     LUA_DECLARE(GetEffectSpeed);
     LUA_DECLARE(SetEffectDensity);
     LUA_DECLARE(GetEffectDensity);
+
+    static bool FxCreateParticle(std::string strParticle, CVector vecPosition, CVector vecDirection, float fR, float fG, float fB, float fA, std::optional<bool> bRandomizeColors, std::optional<int> iCount, std::optional<float> fBrightness, std::optional<float> fSize, std::optional<bool> bRandomizeSizes, std::optional<float> fLife);
```

Comment:
`std::optional<int> iCount `-> `std::uint32_t` or `uint32_t` (see which you prefer).